js/main.js

```diff
@@ -442,16 +441,19 @@ function profit(crop) {
                 var kegModifier = getKegModifier(crop);
                 var caskModifier = getCaskModifier();
                 var dehydratorModifier = getDehydratorModifier(crop);
+				var kegPrice = 0;
                 if (options.produce == 1) {
-                    netIncome += itemsMade * (crop.produce.jarType != null ? crop.produce.jarType : options.skills.arti ? (crop.produce.price * 2 + 50) * 1.4 : crop.produce.price * 2 + 50);
+                    netIncome += itemsMade * (crop.produce.jarType == null ? 0 : options.skills.arti ? (crop.produce.price * 2 + 50) * 1.4 : crop.produce.price * 2 + 50);
                 }
                 else if (options.produce == 2) {
-                    netIncome += itemsMade * (crop.produce.kegType != null && options.aging != "None" ? crop.produce.price * kegModifier * caskModifier : crop.produce.price * kegModifier);
+					kegPrice = crop.produce.keg != null ? crop.produce.keg * kegModifier : crop.produce.kegType != null && options.aging != 0 ? crop.produce.price * kegModifier * caskModifier : crop.produce.price * kegModifier;
+                    netIncome += options.skills.arti ? itemsMade * (kegPrice * 1.4) : itemsMde * kegPrice;
```

Comment:
To add to this, currently the calculation for coffee is wrong, because it calculates the `netIncome` by multiplying the number of beans produced by the price of a single cup, ignoring the fact that 5 beans are necessary for a single coffee to be made, but it would be a separate issue that can be fixed once the general pricing logic is correct.